### ISSUE TITLE
A11y description of menu bar is lacking in details

### ISSUE THREAD

**Issue by schidlo** [mate-desktop/mate-panel#1490]
On Fedora 42 MATE spin when you start Orca screen reader, go to the top bar and move to menu bar (the menu containing three submenus) using tab button. It just says "manu bar." This isn't too useful for Orca's users. It doesn't contain any information on how to use that bar just with keayboard and what it contains - I mean what menus are there, what to expect from such menu. This makes it difficult and confusing for such users.

How to reproduce:
1. Turn Orca on by pressing alt+f2 and writing "orca"
2. Go to top bar with Ctrl+Alt+Tab
3. Use tab to move to select Menu bar

Version of component:
mate-panel-1.28.4-1.fc42.x86_64

Other notes:
There is similar problem with sound widget which doesn't have any a11y info at all. I'll fill that elsewhere.

**Comment by vojtapolasek** [comment]
Thank you @schidlo  for reporting this. For context, I discovered it while building a distro for blind users.
You can find it while performing test scenario  TC 15 described at 
https://github.com/vojtapolasek/vojtux/blob/master/TESTING.md
